### ISSUE TITLE
Is it possible to set the starting div "image"

### ISSUE THREAD

**Issue by ironbone** [zemax/jquery-splitFlap#1]
I like this plugin. I think it would be nice to be able to set the starting and the ending situation. 
Now everything works nice when we call the functionality when document is ready. But when I want to start it (for example) on click, first I have to set the final situation in my div and the user see it first  before and then after the animation one more time.

**Comment by zemax** [comment]
I made some changes and cleanup (and a Readme)
Tell me if the new textInit and autoplay options fit your needs ?
cf http://htmlpreview.github.io/?https://github.com/zemax/jquery-splitFlap/blob/master/demo/index.html

**Comment by ironbone** [comment]
It is much more better.
If I can suggest something more. Maybe it could be more useful to be able to set also (or only) the final text in JavaScript. I know I can do it directly by setting the div content. But with this functionality I will just create an empty div and set all the properties from JavaScript at th same time.
At this moment when I want to start a new animation first I have to set the content of a div to the final text first (at this moment the user can see the final text). Then using JavaScript I set the initial text - so now the user do not see the final text. 
When I have the possibility to set (even only) the final text. I will set the init text, directly or using textInit, so the user see the initial text, then I set the final text (the user do not see it at this moment) and then I start the animation.
